.circleci/config.yml

```diff
@@ -3,50 +3,76 @@ version: 2
 jobs:
   build:
     docker:
-      - image: singledigital/bay-ci-builder:4.x
+      - image: singledigital/ci-buildx:4.x
         environment:
           GOSS_FILES_STRATEGY: cp
           DOCKER_VERSION: 20.10.7
           DOCKERHUB_NAMESPACE: bay_ci
+          DOCKER_CLI_EXPERIMENTAL: enabled
     steps:
       - checkout
       - setup_remote_docker:
           version: 20.10.7
       - run:
-          name: Bring the containers up.
-          command: docker-compose up -d
+          name: Prepare the docker context
+          command: |
+            docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+            docker context create dpc
+            docker buildx create --platform linux/amd64,linux/arm64 dpc --name dpc --use
```

Comment:
My Apple M1  says its detected platform is `linux/arm/v8` so maybe need to build for that too? like:

```
docker buildx create --platform linux/amd64,linux/arm64,linux/arm/v8 
```